consent-ws/src/main/java/org/broadinstitute/consent/http/resources/DataSetResource.java

```diff
@@ -247,7 +248,7 @@ public Response updateNeedsReviewDataSets(@QueryParam("dataSetId") String dataSe
     @Produces(MediaType.APPLICATION_OCTET_STREAM)
     @PermitAll
     @Path("/{datasetId}/approved/users")
-    public Response downloadDatasetApprovedUsers(@PathParam("datasetId") String dataSetId) {
+    public Response downloadDatasetApprovedUsers(@PathParam("datasetId") Integer dataSetId) {
```

Comment:
there are 2 different cases in the parameters here and above. above it is dataSetId, here it is datasetId. can we standardize on one?